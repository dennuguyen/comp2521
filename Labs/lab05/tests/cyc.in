c
r 1 2
c
i 1 2
c
q
