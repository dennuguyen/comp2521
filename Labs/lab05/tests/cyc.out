Graph has V=3 and E=3
V    Connected to
--   ------------
0    1 2
1    0 2
2    0 1

> v=0 vis: [0]=0 [1]=0 [2]=0 
edge 0-1
v=1 vis: [0]=1 [1]=0 [2]=0 
edge 1-0
edge 1-2
v=2 vis: [0]=1 [1]=1 [2]=0 
edge 2-0
Cycle

Graph has V=3 and E=3
V    Connected to
--   ------------
0    1 2
1    0 2
2    0 1

> 
Graph has V=3 and E=2
V    Connected to
--   ------------
0    1 2
1    0
2    0

> v=0 vis: [0]=0 [1]=0 [2]=0 
edge 0-1
v=1 vis: [0]=1 [1]=0 [2]=0 
edge 1-0
No cycle

Graph has V=3 and E=2
V    Connected to
--   ------------
0    1 2
1    0
2    0

> 
Graph has V=3 and E=3
V    Connected to
--   ------------
0    1 2
1    0 2
2    0 1

> v=0 vis: [0]=0 [1]=0 [2]=0 
edge 0-1
v=1 vis: [0]=1 [1]=0 [2]=0 
edge 1-0
edge 1-2
v=2 vis: [0]=1 [1]=1 [2]=0 
edge 2-0
Cycle

Graph has V=3 and E=3
V    Connected to
--   ------------
0    1 2
1    0 2
2    0 1

> 